### PR TITLE
Range damage Upgrade

### PR DESCRIPTION
**Pull request by AntonAsbrink**
It increases the range damage :/

### CODE REVIEW

File: StortSpelProjekt/Game/src/Gamefiles/UpgradeManager.cpp [hunk 1]
```diff
@@ -201,6 +202,13 @@ void UpgradeManager::fillUpgradeMap()
 	upgrade->SetID(UPGRADE_KNOCKBACK);
 	// add the upgrade to the list of all upgrades
 	m_AllAvailableUpgrades[upgrade->GetName()] = upgrade;
+
+	// Adding KnockBack Upgrade
```
Comment (kattalainen) [inline StortSpelProjekt/Game/src/Gamefiles/UpgradeManager.cpp]: copy pasta

Comment (AntonAsbrink) [reply StortSpelProjekt/Game/src/Gamefiles/UpgradeManager.cpp]: I would never

**Comment by kattalainen** [comment]
Don't know about the logic behind having an orange as the icon for the upgrade but besides that it seems to be working fine as intended :>

**Review by kattalainen — APPROVED** [review]

**Review by LeoWikstrom — APPROVED** [review]
I like the orange 👍🏽

**Pull request merged**